core/tauri/Cargo.toml

```diff
@@ -96,6 +96,7 @@ encoding_rs = "0.8.31"
 sys-locale = { version = "0.2.3", optional = true }
 tracing = { version = "0.1", optional = true }
 indexmap = { version = "1", features = [ "std", "serde" ], optional = true }
+urlencoding = "2.1.3"
```

Comment:
No need to bring another crate, we already have a similar crate `percent-encoding`